packages/frontend/src/components/MetricQueryData/MetricQueryDataProvider.tsx

```diff
@@ -7,32 +7,27 @@ import {
     isDimension,
     MetricQuery,
     PivotReference,
-    ResultRow,
     TableCalculation,
 } from '@lightdash/common';
-import React, {
-    createContext,
-    FC,
-    useCallback,
-    useContext,
-    useState,
-} from 'react';
+import { createContext, FC, useCallback, useContext, useState } from 'react';
 import { EChartSeries } from '../../hooks/echarts/useEcharts';
 import { useExplore } from '../../hooks/useExplore';
-import { TableColumn } from '../common/Table/types';
 import { EchartSeriesClickEvent } from '../SimpleChart';
 
+export type UnderlyingValue = { raw: unknown; formatted: string };
+export type UnderlyingValueMap = { [fieldId: string]: UnderlyingValue };
+
 type MetricQueryDataConfig = {
-    value: ResultRow[0]['value'];
-    meta: TableColumn['meta'];
-    row: ResultRow;
+    item: Field | TableCalculation | undefined;
+    value: UnderlyingValue;
+    row: UnderlyingValueMap;
```

Comment:
changed table.meta to item to remove `react-table` specific types
changed ResultRow to UnderlyingValueMap to remove `react-table` specific types